### ISSUE TITLE
Tailwind

### ISSUE THREAD

**Issue by mfreeman451** [bachhm-dev/nuxt-advance-chat#3]
I like your code, have you considered using tailwind css and tailwindui.com?

**Comment by bachhm-dev** [comment]
Yes, I had some private projects using tailwind, I will consider using tailwind in some next public projects.
